examples/tools/bevymark.rs

```diff
@@ -238,20 +251,22 @@ fn collision_system(windows: Res<Windows>, mut bird_query: Query<(&mut Bird, &Tr
         if y_pos + HALF_BIRD_SIZE > half_height && y_vel > 0.0 {
             bird.velocity.y = 0.0;
         }
-    }
+    });
 }
 
 fn counter_system(
     diagnostics: Res<Diagnostics>,
     counter: Res<BevyCounter>,
-    mut query: Query<&mut Text>,
+    mut query: Query<&mut Text, With<StatsText>>,
 ) {
     if let Some(fps) = diagnostics.get(FrameTimeDiagnosticsPlugin::FPS) {
         if let Some(average) = fps.average() {
-            for mut text in query.iter_mut() {
-                text.sections[1].value = format!("{}", counter.count);
-                text.sections[3].value = format!("{:.2}", average);
-            }
+            let mut text = query.single_mut();
+
+            text.sections[1].value.clear();
+            write!(text.sections[1].value, "{}", counter.count).unwrap();
```

Comment:
Hmm; I'm conflicted on this change.

On the one hand, it's much better to re-use the string. On the other hand, this is ugly code.

Reply:
I think since this is a benchmark it's worth saving a few allocations per frame.